app/views/pages/badges.html.erb

```diff
@@ -32,7 +32,7 @@
     </p>
     <% if user_signed_in? %>
       <p>
-        <a href="https://dev.to/<%= current_user.username %>">
+        <a href="<%= user_url(current_user) %>">
```

Comment:
There are two code embeds on lines 48 and 56 in this file that have the hardcoded `https://dev.to/` URL in them--is it necessary to change those URLs to `app_url` as well?

Reply:
Yes definitely, thank you!